docs/ENV.md

```diff
@@ -10,6 +10,9 @@ Debug mode on or off. Possible values are `True` and `False`. Do not set to true
 Used by CubeJS. It should be a secret value and shouldn't be added to git.
 You can create a 32-digit secret key from this website: https://generate.plus/en/base64
 
+#### `CUBEJS_SCHEDULED_REFRESH_TIMER`
```

Comment:
are we using pre-aggregations?

Reply:
If this is for refreshing in-memory cache, what should be the value - just `true` or some set number of seconds - as a nice default?

Reply:
how are we triggering scheduled refreshes? this is the recommended process: https://cube.dev/docs/config#options-reference-scheduled-refresh-timer